util/libcrypto.num

```diff
@@ -4524,3 +4524,6 @@ SM2_do_verify                           4465	1_1_1	EXIST::FUNCTION:SM2
 SM2_sign                                4466	1_1_1	EXIST::FUNCTION:SM2
 ERR_load_SM2_strings                    4467	1_1_1	EXIST::FUNCTION:SM2
 SM2_plaintext_size                      4468	1_1_1	EXIST::FUNCTION:SM2
+CONF_ssl_name_find                      4469	1_1_1	EXIST::FUNCTION:
+CONF_ssl_get_cmd                        4470	1_1_1	EXIST::FUNCTION:
+CONF_ssl_get                            4471	1_1_1	EXIST::FUNCTION:
```

Comment:
Exporting functions from libcrypto so libssl can use them just feels so wrong.

Reply:
It solves a probllem that's otherwise a catch 22, so I think it should be allowed.

However, since these are internal by design from the start, they should probably be all lowercase.

Reply:
There is quite a lot of precedent for this. I changed the names to all lowercase.